spec/schemas/_common.query_dsl.yaml

```diff
@@ -1248,8 +1247,9 @@ components:
           type: number
         filter:
           $ref: '#/components/schemas/QueryContainer'
-      required:
-        - model_id
+      anyOf:
```

Comment:
Does this actually work?

Reply:
will try to test is. that is based on the documentation and other uses in the same file. will try to work on a test.

Reply:
Does not work. Removing that validation.